src/components/Nav/Nav.module.scss

```diff
@@ -165,6 +165,9 @@
     text-decoration: none;
     padding: 0.5em;
     margin: 0 -0.5em;
+    &:focus {
+      outline: 2px solid #0070f3;
```

Comment:
would you mind using one of the color variables for this? you should be good to use it in that file since settings is imported at the top

Here looks like you're using $color-primary which resolves to $color-blue-500

for reference https://github.com/colbyfayock/next-wordpress-starter/blob/master/src/styles/settings/_colors.scss#L17

Reply:
I totally missed the scss variables! I have fixed that in my latest commit.